packages/kbn-plugin-generator/sao_template/template/public/components/app.tsx

```diff
@@ -0,0 +1,129 @@
+import { NavigationPublicPluginStart } from '<%= relRoot %>/../src/plugins/navigation/public';
+
+import { PLUGIN_ID, PLUGIN_NAME } from '../../common';
+
+interface <%= upperCamelCaseName %>AppDeps {
+  basename: string;
+  notifications: CoreStart['notifications'];
+  http: CoreStart['http'];
+  navigation: NavigationPublicPluginStart;
+}
+
+export const <%= upperCamelCaseName %>App = ({ basename, notifications, http, navigation }: <%= upperCamelCaseName %>AppDeps) => {
+  // Use React hooks to manage state.
+  const [timestamp, setTimestamp] = useState<string | undefined>();
+
+  const onClickHandler = () => {
+<%_ if (generateApi) { -%>
+    // Use the core http service to make a response to the server API.
+    http.get('/api/<%= snakeCase(name) %>/example').then(res => {
+      setTimestamp(res.time);
+      // Use the core notifications service to display a success message.
+      notifications.toasts.addSuccess(i18n.translate('banana.dataUpdated', {
+        defaultMessage: 'Data updated',
+      }));
+    });
+<%_ } else { -%>
+    setTimestamp(new Date().toISOString());
+    notifications.toasts.addSuccess(PLUGIN_NAME);
+<%_ } -%>
+  };
+
+  // Render the application DOM.
+  // Note that `navigation.ui.TopNavMenu` is a stateful component exported on the `navigation` plugin's start contract.
+  return (
+    <Router basename={basename}>
+      <I18nProvider>
+        <>
+          <navigation.ui.TopNavMenu appName={ PLUGIN_ID } showSearchBar={true} />
+          <EuiPage restrictWidth="1000px">
+            <EuiPageBody>
+              <EuiPageHeader>
+                <EuiTitle size="l">
+                  <h1>
+                    <FormattedMessage
+                      id="<%= camelCase(name) %>.helloWorldText"
+                      defaultMessage="{name}"
+                      values={{ name: PLUGIN_NAME }}
+                    />
+                  </h1>
+                </EuiTitle>
+              </EuiPageHeader>
+              <EuiPageContent>
+                <EuiPageContentHeader>
+                  <EuiTitle>
+                    <h2>
+                      <FormattedMessage
+                        id="<%= camelCase(name) %>.congratulationsTitle"
+                        defaultMessage="Congratulations, you have successfully created a new Kibana Plugin!"
+                      />
+                    </h2>
+                  </EuiTitle>
+                </EuiPageContentHeader>
+                <EuiPageContentBody>
+                  <EuiText>
+                    <p>
+                      <FormattedMessage
+                        id="<%= camelCase(name) %>.content"
+                        defaultMessage="Look through the generated code and checkout the plugin developmment documentation."
+                      />
+                    </p>
+                    <EuiHorizontalRule/>
+                    <p>
+                      <FormattedMessage
+                        id="banana.timestampText"
```

Comment:
leftover 🍌 instead of plugin id

Reply:
One more :) 
https://github.com/elastic/kibana/pull/55281/files#diff-7974db16cc96ebdadc2997afad18f7a8R60